composer.json

```diff
@@ -4,6 +4,7 @@
   "license": "proprietary",
   "require": {
     "php": ">=7.1",
+    "spryker/monitoring": "^2.0.0",
```

Comment:
Why is this now required?

Reply:
I think this is a mistake.

Reply:
It's not a mistake. `spryker/monitoring` contains fixes that are strongly recommended to make this package work properly. Also technically this package can't work without `spryker/monitoring`, therefore listing as dependency will force composer to install it alongside this one.